### DISCUSSION TITLE
Python 的多線程方法

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#542] [Ideas]
### Thread
Python 在執行時，通常是一執行完一個函式個後才會接下去處理第二個，然而藉由使用 threading 內建函式庫是一個相當方便的函式庫，不僅可以讓原本同步的執行變成非同步，大幅減少工作時間，並且可以設置不同的函數觸發事件或執行間隔產生各種變化。
![螢幕擷取畫面 2022-12-27 201746](https://user-images.githubusercontent.com/66071134/209665655-86560d3e-f70a-42d7-bbdb-d18759d2acd8.jpg)

https://steam.oxxostudio.tw/category/python/library/threading.html

**Comment by awkward-willy** [comment]
Multi-threading 很適合像是爬蟲等需要時間等待 request 的活動使用，也是在使用 Python 開發過程中需要學習使用的技巧之一！

**Comment by Ben6uo** [comment]
感覺阿 這個多線程是一個很酷的東西

**Comment by williamyen0623** [comment]
多線程的運作方式孕育了很多很有價值的演算法和應用，只能說python實在太強大了！

**Comment by HsuChen88** [comment]
這或許能解決我們期末專題遇到的難題呢!
